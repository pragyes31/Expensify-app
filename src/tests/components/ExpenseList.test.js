import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses';
import {ExpenseList} from '../../components/ExpenseList'

test('should render expense list with expenses', () => {
    const wrapper = shallow(<ExpenseList expenses={expenses} />)
    expect(wrapper).toMatchSnapshot();
})

test('should render expense list with no expenses', () => {
    const wrapper = shallow(<ExpenseList expenses={[]} />)
    expect(wrapper).toMatchSnapshot();
})