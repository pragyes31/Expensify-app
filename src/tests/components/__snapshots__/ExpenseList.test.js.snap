// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list with expenses 1`] = `
<div>
  <ExpenseListItem
    amount={313}
    createdAt={-432000000}
    description="Movie"
    id={0}
    key="0"
    note="badla"
  />
  <ExpenseListItem
    amount={300}
    createdAt={345600000}
    description="Fuel"
    id={1}
    key="1"
    note="petrol"
  />
  <ExpenseListItem
    amount={114}
    createdAt={0}
    description="Food"
    id={2}
    key="2"
    note="dosa"
  />
</div>
`;

exports[`should render expense list with no expenses 1`] = `
<div>
  <h1>
    No expenses
  </h1>
</div>
`;
